README.md

```diff
@@ -21,4 +21,44 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
 
 ### Quick Start
 
-Go [here](docs/README.md) for an example of how to get up and going with the cluster api using vSphere.
\ No newline at end of file
+Go [here](docs/README.md) for an example of how to get up and going with the cluster api using vSphere.
+
+### Where to get the vSphere provider container
+
+The containers for this provider are currently hosted at `gcr.io/cnx-cluster-api/`.  Each release of the
+container are tagged with the release version appropriately.
+
+| release version | container url |
+| --- | --- |
+| 0.1 | gcr.io/cnx-cluster-api/vsphere-cluster-api-provider:v0.1 |
+| 0.2 | gcr.io/cnx-cluster-api/vsphere-cluster-api-provider:0.2 |
```

Comment:
v0.2? or v0.1 -> 0.1, i am not sure, should be consistent.

Reply:
this should be consistent across providers, at the very least consistent across our releases.